src/dashboard/get_chart_data/get_chart_data.py

```diff
@@ -165,9 +165,10 @@ def _build_query(query_params: dict, filter_groups: list, path_params: dict) ->
                 raise errors.AggregatorFilterError(
                     f"Invalid filter type {filter_config[1]} requested."
                 )
-
-        inline_configs.append(config_params)
-        none_configs.append(none_params)
+        if config_params != []:
```

Comment:
`if config_params:` might be more natural, unless there's a reason you want to append a `None` value.